src/stylesheets/components/_footer.scss

```diff
@@ -272,3 +272,43 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social-link {
+  $hit-area: 4.5rem; // used for height and width
```

Comment:
Is there a reason we're using 4.5rem instead of 4.4rem - which is what Apple defines?